src/pages/mypage/index.tsx

```diff
@@ -1,20 +1,33 @@
-import Cookies from "cookies";
-import Iron from "@hapi/iron";
-import { GetServerSideProps, NextPage } from "next";
-import React, { useContext } from "react";
-import Router from "next/router";
+import Cookies from 'cookies';
+import Iron from '@hapi/iron';
+import { GetServerSideProps, NextPage } from 'next';
+import React, { useContext, useEffect, useState } from 'react';
+import Router from 'next/router';
 import {
   CenterHorizon,
   CustomButton,
   CustomText,
-} from "../../components/CustomMUI/CustomUI";
-import classes from "./index.module.css";
-import { LoggedInContext } from "../../components/login/IsLoggedIn";
+} from '../../components/CustomMUI/CustomUI';
+import classes from './index.module.css';
+import { LoggedInContext } from '../../components/login/IsLoggedIn';
+import {
+  Avatar,
+  Modal,
+  Typography,
+  ImageList,
+  ImageListItem,
+  Button,
+  ListItemButton,
+  ListItemText,
+  List,
+} from '@mui/material';
+import Box from '@mui/material/Box';
+import Image from 'next/image';
 export const getServerSideProps: GetServerSideProps = async (context) => {
   const cookies = new Cookies(context.req, context.res);
-  const sessionStr = cookies.get("session");
+  const sessionStr = cookies.get('session');
```

Comment:
You are making a lot of changes to this file that are pure "formatting", without changing the logic of the code. This is probably because you as a team have not agreed on what formatting to use, and some of you have automatic formatting in your editor that overwrites the formatting of entire files even if you only change a single line.

Making changes like these increase the risk of merge conflicts, and makes it more difficult for readers of the code to understand what changes were made by recent commits.